cmd/poller/exporter/exporter.go

```diff
@@ -23,7 +23,7 @@ type Exporter interface {
 	Init() error      // initialize exporter
 	GetClass() string // the class of the exporter, e.g. Prometheus, InfluxDB
 	GetName() string  // the name of the exporter instance
-	// Name is different from Class, since we can have multiple instances of the same Class
+	// GetExportCount Name is different from Class, since we can have multiple instances of the same Class
```

Comment:
GetExportCount goes with the line below, but the rest of this comment goes with the line above